src/components/Dashboard/Form/Button/Button.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../../../scss/variables";
+@import "../../../../scss/mixins";
+
+.Button{
+
+  width: 129px;
```

Comment:
We probably don't want set width of buttons. Ideally they would just have padding and their width should be dependent on content. For now, these buttons won't accommodate `Very long text someone can use`

Reply:
Done